sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -0,0 +1,43 @@
+
+# Azure Ai Contentsafety client library for Python
+<!-- write necessary description of service -->
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install azure-ai-contentsafety
```

Comment:
You can just use pip install azure-ai-contentsafety

Reply:
Fxied.